Cargo.toml

```diff
@@ -4,12 +4,12 @@ version = "2.7.1"
 authors = [
 	"Jonathan 'theJPster' Pallant <github@thejpster.org.uk>",
 	"42 Technology Ltd <jonathan.pallant@42technology.com>",
-	"Dion Dokter <diondokter@gmail.com>"
+	"Dion Dokter <diondokter@gmail.com>",
 ]
 edition = "2021"
 description = "Rust bindings to the Nordic nRF91* Socket Library."
 readme = "README.md"
-license-file = "LICENCE.md"
+license-file = "LICENSE.md"
```

Comment:
Why the change? It was correct

Reply:
That was just the spelling, this does not necessarily need to be in this pull request.